### ISSUE TITLE
[Request] Create uninstall command for v2

### ISSUE THREAD

**Issue by stevenrick** [empiricaly/empirica#158]

**Comment by npaton** [comment]
Agreed, this is a good idea. We will implement this. Empirica's footprint goes beyond the main command line since there's a system to manage multiple versions and the nodejs runtime.
